packages/core/admin/admin/src/pages/App/index.js

```diff
@@ -77,7 +77,7 @@ function App() {
               body: JSON.stringify({
                 event: 'didInitializeAdministration',
                 uuid,
-                deviceId,
+                deviceId: `web-fingerprint-${deviceId}`,
```

Comment:
Can we just change the getUID function to add the prefix instead. No need to leak this abstraction in the code here

Reply:
No worries, I'll rework this 👍